app/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ android {
 }
 
 dependencies {
-    implementation("com.google.android.material:material:1.6.0")
```

Comment:
What is this package you are removing?

Reply:
materials design, I added it when I was trying to figure out rounded corners for my first pull request and never used it